bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/quickaccess/providers/CommandProvider.java

```diff
@@ -117,6 +154,33 @@ private void retrieveCommand(final String currentCommandId) {
 		}
 	}
 
+	private boolean filteredWithActivitySupport(ParameterizedCommand pc) {
```

Comment:
`filter` is generally an ambiguous name and the more frequent usage in Java code wiht `Stream` has the opposite meaning of what you're using here (Streams "keep" filtered content, you want to exclude it), here using `excludeWithActivitySupport` would be better.

Reply:
Fixed.